src/main/java/yapmeister/task/ToDo.java

```diff
@@ -0,0 +1,38 @@
+package yapmeister.task;
+
+/**
+ * Represents a Todo Task
+ * @author BlazeChron
+ */
+public class ToDo extends Task {
+
+    /**
+     * Creates a Todo Task
+     * @param taskName Name of the task.
+     */
+    public ToDo(String taskName) {
+        super(taskName);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String exportString() {
```

Comment:
After looking through your code, almost all of your method names follow the code quality guidelines. Nicely done!